apps/logging/serializers.py

```diff
@@ -210,9 +209,8 @@ def to_dict(self):
 
 class Response:
     request_class = None
-    resp = None
 
-    def __init__(self, response):
+    def __init__(self, response, **kwargs):
```

Comment:
@JFU-GIT I don't think the `**kwargs` argument is needed here and can be removed?

Reply:
agree

Reply:
extra **kwargs removed